### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by FRadice1977** [NewtonAcademy/FrankJunior#1]
Hi,

I am having trouble getting this set up. Could I request some help? 

Francesca.

**Comment by samwarfield** [comment]
What part are you stuck at?

Sam

**Comment by FRadice1977** [comment]
I have uploaded to the server with an /api file extension.  I get stuck on step 2. Saving a DB outside the root. I'm not sure what you mean there.

**Comment by samwarfield** [comment]
The database should be saved somewhere where its secure... Not in the /api folder... Root is probably where your /api folder is...

inside the index.php file is a comment

The following require imports a php file which is stored outside our website's root
that defines a functiuon that creates and returns a database connection. We
store this information outside the web root for security reasons. The file's contents
might look something like this:
<?php
function db() {
$host = "localhost";
$user = "www-data";
$password = "W372Kh7JEbQf6Nc";
$database = "frank_jr";
return mysqli_connect($host, $user, $password, $database);
}
?>

**Comment by FRadice1977** [comment]
Ok. I tried running the index.php file on an external database created on the web server. But I received many errors. If my database name is sitesear_frank_jr do I have to change the syntax in the index.php file? Can you think of other changes that need to be made to the index.php when importing to database?

**Comment by samwarfield** [comment]
If you created a file sitesear_frank_jr.php inside /var/www/

The php file you created needs to match

require '/var/www/frank_jr-db.php';

so change it to

require '/var/www/sitesear_frank_jr.php';

**Comment by samwarfield** [comment]
This is part of a program at Newton Academy. There is another document that walks you through it. Are you a student at Newton Academy?

**Comment by FRadice1977** [comment]
No. I am an inexperienced web developer. Do you have a course on Rest API's for trapping data?

**Comment by samwarfield** [comment]
Checkout newtonacademy.org

**Comment by FRadice1977** [comment]
I can't seem to find prices on the website. Can you post a link with those details?

**Comment by samwarfield** [comment]
It is a complete course for building iOS apps and not specific to web development. I would probably take it for learning iOS and not web development. You should start off with some beginner courses udemy.com if you are just wanting just web dev... and see how you do. If they are too advanced. Start a beginning computer science course from somewhere.
